Generator/AssociationTypeGenerator.php

```diff
@@ -21,6 +21,8 @@
  */
 class AssociationTypeGenerator implements GeneratorInterface
 {
+    const TYPE = 'association_types';
+
     const ASSOCIATION_TYPES_FILENAME = 'association_types.csv';
```

Comment:
Just wondering how do you plan to make the project evolve with xlsx. Should this generator be format agnostic ? so that afterwards you can write csv or xlsx files ? Maybe for another PR but I'd like to know your take on that point.